libpod/logs/log.go

```diff
@@ -120,11 +122,16 @@ func getTailLog(path string, tail int) ([]*LogLine, error) {
 func (l *LogLine) String(options *LogOptions) string {
 	var out string
 	if options.Multi {
-		cid := l.CID
-		if len(cid) > 12 {
-			cid = cid[:12]
+		if options.UseName {
+			cname := l.CName
+			out = fmt.Sprintf("%s ", cname)
```

Comment:
```suggestion
			out = l.CName + " "
```
The suggested change is faster and is more explicit about what you want. Also removes a temporary variable.

Reply:
https://github.com/containers/libpod/pull/5468